src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -132,8 +134,10 @@ public boolean clickEditInstructorLink(int instrNum) {
             WebElement saveButton = browser.driver.findElement(By.id("btnSaveInstructor" + instructorNum));
             waitForElementVisibility(saveButton);
             
-            WebElement editInstructorNameTextBox = browser.driver.findElement(By.id("instructorname" + instructorNum));
-            WebElement editInstructorEmailTextBox = browser.driver.findElement(By.id("instructoremail" + instructorNum));
+            WebElement editInstructorNameTextBox = browser.driver.findElement(By.id(Const.ParamsNames.INSTRUCTOR_NAME
+                                                                                    + instructorNum));
```

Comment:
Same comment as above regarding how `Const.ParamsNames` refer to the `name` attribute and not the `id` attribute